addOns/exportreport/src/main/java/org/zaproxy/zap/extension/exportreport/export/ReportExportPDF.java

```diff
@@ -0,0 +1,1100 @@
+                            alert.getMessage().getResponseHeader().toString(),
+                            textInsertionPoint);
+            textInsertionPoint = addText(alertTextFormatting, " ", textInsertionPoint);
+        }
+
+        // Response Body
+        if (detailsToInclude.contains(
+                extensionExport
+                        .getMessages()
+                        .getString("exportreport.details.responsebody.label"))) {
+            textInsertionPoint = addText(alertLabelFormatting, labelRespBody, textInsertionPoint);
+            textInsertionPoint =
+                    addText(
+                            alertTextFormatting,
+                            alert.getMessage().getResponseBody().toString(),
+                            textInsertionPoint);
+            textInsertionPoint = addText(alertTextFormatting, " ", textInsertionPoint);
+        }
+
+        textInsertionPoint = addText(alertLabelFormatting, labelURLs, textInsertionPoint);
+
+        // TODO: binary data (Base64 decoded data is the only example I can find) flows onto the
+        // margin..
+        // can we do something about it??
+
+        // for each alert within this category
+        for (int i = 0; i < alerts.size(); i++) {
+            Alert alertAux = alerts.get(i);
+
+            // output the URL, and parameter information for each alert for this category
+            textInsertionPoint =
+                    addText(
+                            alertTextFormatting,
+                            (i + 1) + "-" + alertAux.getUri(),
+                            textInsertionPoint);
+
+            if (!alertAux.getParam().isEmpty()) {
+                textInsertionPoint =
+                        addText(
+                                alertTextFormatting,
+                                labelParameter + ": " + alertAux.getParam(),
+                                textInsertionPoint);
+            }
+            if (alertAux.getAttack() != null && !alertAux.getAttack().isEmpty()) {
+                textInsertionPoint =
+                        addText(
+                                alertTextFormatting,
+                                labelAttack + ": " + alertAux.getAttack(),
+                                textInsertionPoint);
+            }
+            if (alertAux.getEvidence() != null && !alertAux.getEvidence().isEmpty()) {
+                textInsertionPoint =
+                        addText(
+                                alertTextFormatting,
+                                labelEvidence + ": " + alertAux.getEvidence(),
+                                textInsertionPoint);
+            }
+
+            // Not added if not selected as detail to include
+            if (detailsToInclude.contains(
+                            extensionExport
+                                    .getMessages()
+                                    .getString("exportreport.export.message.pdf.otherinfo"))
```

Comment:
The key needs to be changed to `details`.

Reply:
Updated all `exportreport.export.message.pdf` keys to `exportreport.details.[detail_name].label` to match the other details keys.